apps/user/src/components/form/apps/성적입력/AttendanceCalculator/AttendanceCalculator.hooks.tsx

```diff
@@ -0,0 +1,91 @@
+import { ChangeEventHandler, useEffect, useState } from 'react';
+
+interface Attendance {
+    absenceCount: number;
+    latenessCount: number;
+    earlyLeaveCount: number;
+    classAbsenceCount: number;
+}
+
+export interface AttendanceInfo {
+    attendance1: Attendance;
+    attendance2: Attendance;
+    attendance3: Attendance;
+}
+
+const useInput = () => {
+    const [attendanceInfo, setAttendanceInfo] = useState<AttendanceInfo>({
+        attendance1: {
+            absenceCount: 0,
+            latenessCount: 0,
+            earlyLeaveCount: 0,
+            classAbsenceCount: 0,
+        },
+        attendance2: {
+            absenceCount: 0,
+            latenessCount: 0,
+            earlyLeaveCount: 0,
+            classAbsenceCount: 0,
+        },
+        attendance3: {
+            absenceCount: 0,
+            latenessCount: 0,
+            earlyLeaveCount: 0,
+            classAbsenceCount: 0,
+        },
+    });
+
+    const handleAttendance1InfoDataChange: ChangeEventHandler<HTMLInputElement> = (e) => {
+        const { name, value } = e.target;
+
+        const updatedAttendance1: Attendance = {
+            ...attendanceInfo.attendance1,
+            [name]: +value,
```

Comment:
```suggestion
            [name]: Number(value),
```

뒤에 유지보수를 하는 입장에서 +가 숫자형 변환 연산자인지 알기 힘들것같아요 좀 더 명시적인 Number가 나을것같아요!